starter-core/src/main/java/io/micronaut/starter/feature/json/JsonSchemaGeneratorFeature.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2017-2024 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.json;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.core.util.StringUtils;
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.build.dependencies.MicronautDependencyUtils;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.Feature;
+import jakarta.inject.Singleton;
+
+@Requires(property = "micronaut.starter.feature.json.schema.generator.enabled", value = StringUtils.TRUE, defaultValue = StringUtils.TRUE)
+@Singleton
+public class JsonSchemaGeneratorFeature implements Feature {
+
+    public static final String NAME = "json-schema-generator";
+
+    private static final Dependency JSON_SCHEMA_GENERATOR_DEPENDENCY = MicronautDependencyUtils.jsonSchemaDependency()
+            .artifactId("micronaut-json-schema-generator")
+            .compile()
+            .build();
+    private static final Dependency JSON_SCHEMA_ANNOTAIONS_DEPENDENCY = MicronautDependencyUtils.jsonSchemaDependency()
+            .artifactId("micronaut-json-schema-annotations")
+            .compile()
+            .build();
+
+    @Override
+    public String getName() {
+        return NAME;
+    }
+
+    @Override
+    public String getTitle() {
+        return "JSON Schema Generator";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Adds Source Generator from JSON Schema to a Micronaut Application";
```

Comment:
Could this be "Adds capability to generate source classes from JSON schema definitions"?

Also, could you please change the description of the `JsonSchemaFeature`, so the difference is clear.